### PR TITLE
Esckan - 10 & 13 . Header and About dialog. UI skeleton

### PR DESCRIPTION
**Pull request by vidhya-metacell**
Screen Shot 2024-02-13 at 2 50 54 PM
Screen Shot 2024-02-13 at 2 51 02 PM

### CODE REVIEW

**Comment by zsinnema** [comment]
@vidhya-metacell the eslint check failed. Could you have a look and fix them? thanks!

**Comment by vidhya-metacell** [comment]
@zsinnema when i run "npm run lint", i see no errors on the terminal.

**Comment by zsinnema** [comment]
Thanks for checking @vidhya-metacell , did you look into the logs of the failing jobs? f.e. https://github.com/MetaCell/sckan-explorer/actions/runs/7887539873/job/21523013039?pr=4
This one shows that the lock file is outdated

**Comment by vidhya-metacell** [comment]
@zsinnema updated the file now.

File: src/components/common/About.tsx [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+      <DialogContent dividers>
+        <Box>
+          <Typography gutterBottom variant='h6'>About SCKAN Explorer</Typography>
+          <Typography gutterBottom>
+            A key component of the SPARC Program is SCKAN. It is a semantic store housing a comprehensive knowledge base of autonomic nervous system (ANS) and peripheral nervous system (PNS) nerve to end organ connectivity. Connectivity information is derived from SPARC experts, SPARC data, literature and textbooks. SCKAN supports reasoning and offers powerful query and visualization capabilities.
+          </Typography>
+          <Typography variant='button'>Learn more about SCKAN</Typography>
+        </Box>
+        <Box>
+          <Typography gutterBottom variant='h6'>Owner</Typography>
```
Comment (zsinnema) [inline src/components/common/About.tsx:50]: suggestion @vidhya-metacell : create a small component for this <box>..</box>?

**Review by zsinnema — CHANGES_REQUESTED** [review]

File: src/components/common/About.tsx (hunk 1 above)
Comment (vidhya-metacell) [reply src/components/common/About.tsx]: @zsinnema updated

**Review by zsinnema — APPROVED** [review]

**Pull request merged**